### ISSUE TITLE
Images have unexpected tint on iOS

### ISSUE THREAD

**Issue by j-roskopf** [Skeptick/libres#40]
Hey!

When displaying images on iOS, there seems to be a tint applied that does not happen on Android.

For example, I have a file in the project named `dog1_(orig).png`

Here is what it looks like rendering in Android Studio:
<img width="640" alt="Screenshot 2023-11-04 at 2 39 30 PM" src="https://github.com/Skeptick/libres/assets/7951665/4daa5dc3-13dc-41a6-8964-709751ab3a59">

Here is how it renders on iOS:
<img width="146" alt="Screenshot 2023-11-04 at 2 40 03 PM" src="https://github.com/Skeptick/libres/assets/7951665/fca4f49b-d995-4d29-97d9-2cc23a476589">

Here is how it renders on Android:
<img width="136" alt="Screenshot 2023-11-04 at 2 40 25 PM" src="https://github.com/Skeptick/libres/assets/7951665/2395d574-08b6-490e-a2cf-0ec89114c651">

Similarly, I have another icon named `full_availability_(orig).png` and here is how it renders in Android Studio:
<img width="600" alt="Screenshot 2023-11-04 at 2 40 54 PM" src="https://github.com/Skeptick/libres/assets/7951665/fc22b1b4-c6ec-4e1a-b435-e54bbc96a895">

Here is that image rendering on Android:
<img width="137" alt="Screenshot 2023-11-04 at 2 41 34 PM" src="https://github.com/Skeptick/libres/assets/7951665/0003d8c3-5f2e-467d-bb5a-826faddf657a">

Here is that image rendering on iOS:
<img width="149" alt="Screenshot 2023-11-04 at 2 42 01 PM" src="https://github.com/Skeptick/libres/assets/7951665/ed2ba230-d457-4084-9978-ff91fe1f129f">

Here is sample code of how I am displaying the last image:

```
            Image(
                painter = MainResImages.full_availability.painterResource(),
                contentDescription = "availability",
                modifier = Modifier.size(32.dp),
            )
```

Using version 1.2.0 of the project.

Is there some additional configuration I need on my side that I missed? Or does it appear to be an actual issue?

Thanks!

**Comment by Skeptick** [comment]
Hi. What version of compose are you using?
Please provide the original of one of these images, I will try to reproduce it

**Comment by j-roskopf** [comment]
Hey! Thanks for the quick reply. Using `1.5.10` of Compose.

Here is the image: https://drive.google.com/file/d/1xltTYhLvW4zcNwvYnHcWX10YqSTlbrkh/view?usp=sharing

**Comment by Skeptick** [comment]
Yes, this is bug. I published version 1.2.1 with fix

**Comment by j-roskopf** [comment]
Updated to 1.2.1 and verified images look as expected on iOS. Thanks!
